src/sentry/search/snuba/backend.py

```diff
@@ -247,20 +233,6 @@ def query(
             "active_at": ScalarCondition("active_at"),
         }
 
-        message = [
-            search_filter for search_filter in search_filters if search_filter.key.name == "message"
-        ]
-        if message and message[0].value.raw_value:
-            message = message[0]
-            # We only support full wildcard matching in postgres
-            if is_postgres() and message.value.is_wildcard():
-                group_queryset = message_regex_filter(group_queryset, message)
-            else:
-                # Otherwise, use the standard LIKE query
-                qs_builder_conditions["message"] = QCallbackCondition(
-                    lambda message: Q(Q(message__icontains=message) | Q(culprit__icontains=message))
```

Comment:
Is no longer searching across `culprit` going to matter much to customers? I'm not sure we expose `culprit` to them as a field.

Reply:
Not sure if I am qualified to answer. `culprit` looks like it is in the clickhouse sentry_local table so that could be used if we need it.